lesson13/index.js

```diff
@@ -45,17 +50,31 @@ const createList = async ({data})=> {
   ul.appendChild(fragment);
 }
 
+const appendToModal = ()=>{
```

Comment:
IMO
スタイルの問題だから一貫していたらいいかなと思ったのですがそうでもなさそうなので
```suggestion
const appendToModal = () => {
```

Reply:
ESLint入れねばですね (>_<)
修正しました！

70998c0